debian/generate-dockerfile.sh

```diff
@@ -75,7 +75,9 @@ for arch in $archs; do
 
 	# Tini
 	tiniBinary="tini$TINI_VERSION.linux-$arch.tar.gz"
+	IFS='%'
 	tiniChecksum="TINI_$arch" && tiniChecksum=$(eval echo \$$tiniChecksum)
```

Comment:
Setting the IFS is very hacky. You can instead do `tiniChecksum=$(eval echo \"\$$tiniChecksum\")`. Also I don't understand why you don't do `tiniChecksum=$(eval echo \"\$TINI_$arch\")` directly